types/index.d.ts

```diff
@@ -338,7 +338,7 @@ export type MemberAssignment = {
 
 export const AssignerProtocol: {
   MemberMetadata: ISerializer<MemberMetadata>
-  MemberAssignment: ISerializer<MemberAssignment>
+  MemberAssignment: ISerializer<MemberAssignment | null>
```

Comment:
That means MemberAssignment#encode() would need to handle `null` as well now, right?

Reply:
Right! Took me a bit to see what you meant, but yes indeed, it would mean that `encode` would also need to handle receiving null. 

I changed it now so that only the `decode` has a nullable return type.